addons/mrp_subcontracting/models/stock_move.py

```diff
@@ -73,7 +73,7 @@ def action_show_details(self):
         subcontracted product. Otherwise use standard behavior.
         """
         self.ensure_one()
-        if self._subcontrating_can_be_record():
+        if (self._subcontrating_should_be_record() | self._subcontrating_can_be_record()):
```

Comment:
There is any upside to do that instead of
```suggestion
        if self._subcontrating_should_be_record() or self._subcontrating_can_be_record():
```
?

Reply:
Nop it is even better it avoids to call _subcontrating_can_be_record if there is some _subcontrating_should_be_record.
Thanks.